tck/api/src/main/java/org/eclipse/microprofile/metrics/tck/MetricRegistryTest.java

```diff
@@ -97,28 +74,6 @@ public void nameTest() {
         Assert.assertNotNull(metrics.getMetadata("nameTest"));
     }
 
-    @Test
-    @InSequence(2)
-    public void registerTest() {
```

Comment:
not clear why this whole test is being removed instead of just the tests for the metric types we no longer have

Reply:
This test was testing against the _registration_ method that allowed for users to provide an implementation of a Metric. It was using an injected metric instance (the deleted injection points above) and then registering it under a new name.